editor/src/core/shared/template-path.ts

```diff
@@ -196,6 +197,10 @@ export function scenePath(elementPaths: ElementPath[]): ScenePath {
   }
 }
 
+export function staticScenePath(elementPaths: ElementPath[]): StaticScenePath {
+  return scenePath(elementPaths) as StaticScenePath
```

Comment:
This is also dangerous right?

Reply:
I'd advocate for a tiny bit of duplicated code over these kinds of casts.

Reply:
Oops, this function was supposed to take `StaticElementPath[]`. Good catch.

Reply:
Done